server/channels/store/sqlstore/tokens_store.go

```diff
@@ -51,12 +61,18 @@ func (s SqlTokenStore) Delete(token string) error {
 func (s SqlTokenStore) GetByToken(tokenString string) (*model.Token, error) {
 	var token model.Token
 
-	if err := s.GetReplica().Get(&token, "SELECT * FROM Tokens WHERE Token = ?", tokenString); err != nil {
+	query, args, err := s.tokenSelectQuery.
+		Where(sq.Eq{"Token": tokenString}).
+		ToSql()
+
+	if err != nil {
+		return nil, errors.Wrap(err, "could not build sql query to get token store")
+	}
+
+	if err := s.GetReplica().Get(&token, query, args); err != nil {
 		if err == sql.ErrNoRows {
 			return nil, store.NewErrNotFound("Token", fmt.Sprintf("Token=%s", tokenString))
 		}
-
```

Comment:
It looks like these lines were incorrectly removed, changing the semantics of this method when a token can't be found.

Reply:
I added it back, could you run the CI workflow? Also is there a way to run the tests locally without specifying the 50 or so failing tests or running all of them at once?